sass/directives/_carousels.scss

```diff
@@ -121,30 +121,32 @@
 
 @mixin resource-continue-block {
   border-top: $base-border;
-  padding-top: $base-padding / 2;
-  padding-bottom: $base-padding / 2;
+  border-color: $grey-lighter;
+  background-color: $brand-new-grey-lightest;
+  padding-top: $base-font-size / 2;
```

Comment:
I wouldn't use a font size variable here for this, since it doesn't relate to padding it could cause some unwanted side effects if we changed the `$base-font-size`. Instead I would recommend just using `$base-margin`, which is close to what you're getting here (12px vs. 8px)